projects/mercury/src/common/utils/genericUtils.js

```diff
@@ -163,6 +165,16 @@ export const formatDateTime = value => {
     return isToday ? timeFormatter.format(date) : dateFormatter.format(date);
 };
 
+export const renderBooleanIcon = value => {
+    if (value === true || value === 't') {
```

Comment:
probably it would make sense to turn it to lower case first and then compare

Reply:
I changed this part